### PR TITLE
[ExpressionLanguage] Document ExpressionFunction::fromPhp

### PR DESCRIPTION
**Pull request by maidmaid**
cf symfony/symfony#21122

### CODE REVIEW

File: components/expression_language/extending.rst [hunk 1]
```diff
@@ -90,6 +90,17 @@ register.
...
     }
 
+.. tip::
+
+    You can create an expression function from a PHP function with the
+    :method:`Symfony\\Component\\ExpressionLanguage\\ExpressionFunction::fromPhp` static method::
+
+        ExpressionFunction::fromPhp('strtoupper');
+
+    It allows namespaced PHP function where second parameter must be defined to name expression::
```
Comment (xabbuh) [inline components/expression_language/extending.rst]: [...] where the second [...]

Comment (maidmaid) [inline components/expression_language/extending.rst]: Done

**Comment by xabbuh** [comment]
Please also add a `versionadded` directive:

```rst
.. versionadded:: 3.3
    The ``ExpressionFunction::fromPhp()`` method was introduced in Symfony 3.3.
```

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Comment by maidmaid** [comment]
Result: 

screenshot from 2017-02-21 19-06-31

[link to temp doc](http://pr-7517-wxuwdhq-6qmocelev2lwe.eu.platform.sh/components/expression_language/extending.html#using-expression-providers)

**Review by javiereguiluz — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
@maidmaid This should work. The template used to render the preview is not exactly the same as the one used on symfony.com (we don't update it every time small changes are made to the website).

**Comment by maidmaid** [comment]
Cool, thank you for your reviews :)

File: components/expression_language/extending.rst [hunk 2]
```diff
@@ -90,6 +90,21 @@ register.
         }
     }
 
+.. tip::
+
+    You can create an expression function from a PHP function with the
```
Comment (HeahDude) [inline components/expression_language/extending.rst]: "You can create ..." => "To create ..."

**Review by HeahDude — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
Thank you @maidmaid.

**Pull request merged**